HIPS/hip-1.md

```diff
@@ -0,0 +1,54 @@
+---
+hip: <to be assigned>
+title: <HIP title>
+author: <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s), e.g. (use with the parentheses or triangular brackets): FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
+status: Example
+type: <Standard Track | Informational>
+category (*only required for Standard Track): <Core | Interface | Economics | HRC>
```

Comment:
Informational HIPs wouldn't necessarily get "accepted" as more "answered" in my eyes. But agree we should likely give more thought into the differences for v.2 or future updates. 

I think this is fine to move forward with, in the short term.. and this way, we at least know at a top-level if it's standard vs. info. If it's info, then maintainers/developers throughout the community know it won't impact their projects and only need to follow the HIP if they are curious about the details.

Reply:
Ok, checking the EIP and it has a Meta track as well which is really a process track. Should we add a `Process` type?

Reply:
That makes sense. Do you think something like this would be acceptable for the README?

- Process: This type of proposal includes suggested changes to the HIP process itself.

Or is there additional phrasing/information you think would be valuable?

Reply:
No, it's encompasses more than just the HIP process but the process of the entire hedera ecosystem. See the EIP [description](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md#eip-types)

Reply:
That makes sense. Inspired by their description, how does this sound? 

> Process: This type of proposal includes any changes that affect the Hedera development process. Process HIPs are similar to Standard HIPs but modify tools and software beyond just the core Hedera Hashgraph protocol. Proposals of this category are typically more than simple recommendations, often requiring users to pay attention and make subsequent changes to additional software. Because of the impact in the Hedera development process, these proposals often require a community-based consensus. Examples include but are not limited to: developer guidelines, changes to an existing decision-making process, and changes to the tools or environment used in Hedera development.

Reply:
Looks goods

Reply:
Should be resolved in b123da7917e6021be183edc95c234e2de98da1c8